ingestion/examples/sample_data/policies/access_control.json

```diff
@@ -11,46 +11,22 @@
           "name": "update-description",
           "userRoleAttr": "DataSteward",
           "operation": "UpdateDescription",
-          "allow": true
+          "allow": true,
+          "enabled": true
         },
         {
           "name": "update-tags",
           "userRoleAttr": "DataSteward",
-          "operation": "UpdateTags",
-          "allow": true
-        }
-      ]
-    },
-    {
-      "name": "data-consumer-role",
```

Comment:
Remove DataConsumer role for now.

For next release, we'll focus on DataSteward and getting that done correctly

Reply:
DataConsumer could be the default role and all users should have that role assigned to them

Reply:
DataProducer, DataSteward and Admins can be special role

Reply:
+1 Opening another issue for adding more roles (also the default ones from data/roles)

Reply:
#2230